ui/src/app/applications/components/application-deployment-history/application-deployment-history.tsx

```diff
@@ -37,6 +37,12 @@ export const ApplicationDeploymentHistory = ({
                             <br />
                             <Timestamp date={info.deployedAt} />
                         </div>
+                        <div>
+                            <br />
+                            <i className='fa fa-hourglass-half' /> Time to deploy:
+                            <br />
+                            <Duration durationMs={moment(info.deployedAt).diff(moment(info.deployStartedAt)) / 1000} />
```

Comment:
Hm, what will be the outcome of this when there's no `deployStartedAt` value in the revision history, i.e. for syncs happened before this change?

Reply:
Thanks for the catch, fixed it to look like this:
![image](https://user-images.githubusercontent.com/6951919/84187723-1adeda80-aab0-11ea-9ff1-d523f856e1a2.png)